graphql/thunder_media.base.graphqls

```diff
@@ -5,6 +5,17 @@ interface Media {
   name: String
 }
 
+type ImageDerivative {
+  url: String
```

Comment:
We use src instead of url

Reply:
Not possible because Drupal\graphql\Plugin\GraphQL\DataProducer\Entity\Fields\Image\ImageDerivative returns url :(